configs/edison/config.B_low_res_ice_shelves_1696_JWolfe_layout_Edison

```diff
@@ -22,6 +22,13 @@ parallelTaskCount = 1
 # handle 12 simultaneous processes, one for each monthly climatology.
 ncclimoParallelMode = serial
 
+# config file for a reference run to which this run will be compared.  The
+# analysis should have already been run to completion once with this config
+# file, so that the relevant MPAS climatologies already exist and have been
+# remapped to the comparison grid.  Leave this option commented out if no
+# reference run is desired.
+# referenceRunConfigFile = /path/to/config/file
+
```

Comment:
this piece above should be moved to the [runs] section.

Reply:
@xylar: don't forget to move this in the right section for this config file.

Reply:
Thanks!  I think I got this in the latest update.